docs/manual/scala/guide/cluster/DomainModelling.md

```diff
@@ -34,25 +34,34 @@ In Akka Typed, it's not possible to return an exception to the caller. All commu
 ```scala
 // Replies
 sealed trait Confirmation
-sealed trait Accepted extends Confirmation
-case object Accepted extends Accepted
+case object Accepted extends Confirmation
 final case class Rejected(reason: String) extends Confirmation
-final case class CartState(items: Map[String, Int], status: String)
+final case class ShoppingCartSummary(items: Map[String, Int], checkedOut: Boolean)
 
 // Commands
 sealed trait ShoppingCartCommand
-final case class UpdateItem(productId: String,
-                            quantity: Int,
+
+final case class AddItem(itemId: String,
+                         quantity: Int,
+                         replyTo: ActorRef[Confirmation])
+  extends ShoppingCartCommand
+
+final case class RemoveItem(itemId: String,
                             replyTo: ActorRef[Confirmation])
   extends ShoppingCartCommand
 
+final case class AdjustItemQuantity(itemId: String,
+                                    quantity: Int,
+                                    replyTo: ActorRef[Confirmation])
+  extends ShoppingCartCommand
```

Comment:
(nitpick to address on a separate PR)

Not sure what the purpose of `Adjust...` is. I assume it's the idempotent equivalent to `AddItem` and `RemoveItem` (which are not idempotent). If so, Then I'd stick to using only the idempotent command.

Reply:
They are all idempotent. You send twice the same you get the same result. 

But you made me realise one thing, you can also adjust the quantity by sending an `AddItem` command.

Reply:
Ok, I will what you mean. They are not idempotent in the sense that they do produce an event, but they state stays the same.

Reply:
but to answer your question, `AdjustItemQuantity` is to make it explicit the intention. You have the idem already, but you want to change the quantity. Nothing else.